fees/fluid/fees.ts

```diff
@@ -87,4 +248,17 @@ export const getFluidDailyFees = async ({ api, fromApi, toApi, getLogs, createBa
   }
 
   return dailyFees;
+};
+
+export const getFluidDailyFees = async (options: FetchOptions) => {
+  // fetch all operate logs at liquidity layer at once
+  const liquidityOperateLogs = await options.getLogs({ target: LIQUIDITY, onlyArgs: true, topics: [TOPIC0.logOperate], eventAbi: EVENT_ABI.logOperate, flatten: true, skipCacheRead: true });
```

Comment:
this getLogs fetch takes awfully long for me when running it locally. maybe because of RPC? fetching them all at once and then filtering where it is actually needed, seems to have a net positive effect vs filtering via topics, see commit https://github.com/DefiLlama/dimension-adapters/pull/2472/commits/19859ee2b7ba6eec7baaa8e40350159ffa03d2b7